woocommerce/woocommerce-ios
# Shipping Labels M1: analytics

Fixes #2930 

## Why

This PR tracks events for shipping labels M1!

## Changes

- Added `WooAnalyticsStat` cases for new shipping labels M1 events
- Added two `WooAnalyticsEvent` functions for events with properties: `shipmentTrackingMenuAction` and `shippingLabelsAPIRequest`
- Implemented event tracking in corresponding places, and added test cases when possible

## Testing

- Go to the orders tab
- Tap on an order with at least a non-refunded shipping label —> after syncing, should see `🔵 Tracked shipping_label_api_request, properties: [AnyHashable("action"): "success”, …]` in the console and a shipping label package card should appear
- In “Tracking number” row in the shipping label package card, tap the ellipsis menu
- Tap “Copy tracking number” in the action sheet or popover —> should see `🔵 Tracked shipment_tracking_menu_action, properties: [AnyHashable("action"): "copy”, …]` in the console
- In “Tracking number” row in the shipping label package card, tap the ellipsis menu again
- Tap “Track shipment” in the action sheet or popover —> should see `🔵 Tracked shipment_tracking_menu_action, properties: [AnyHashable("action"): "track”, …]` in the console
- Tap “Reprint Shipping Label”
- Tap “Print Shipping Label” —> should see `🔵 Tracked shipping_label_print_requested` in the console
- Tap the ellipsis menu at the top of the shipping label package card (to the right of “Package #”)
- Tap “Request a Refund”
- Tap “Refund Label ($#)” —> should see `🔵 Tracked shipping_label_refund_requested` in the console
- Go offline
- Go back to the orders tab and tap on the same order —> after syncing, should see `🔵 Tracked shipping_label_api_request, properties: [AnyHashable("action"): “failed”, …]` in the console and a shipping label package card should appear

Update release notes:
